.github/pull_request_template.md

```diff
@@ -0,0 +1,145 @@
+
+
```

Comment:
When we add a template anyway, maybe we should also have a section on the PR itself, e.g.

```
# Description

<brief description of PR>

Closes #ISSUE_CLOSED_BY_PR
```

Reply:
Added in 6c75d54dd221700164ceb40b85a729e3547f8533.